app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<h1>Task List</h1>
+
+<table>
+  <tr>
+    <th>
+      Name
+    </th>
+    <th>
+      Owner
+    </th>
+    <th>
+      Status
+    </th>
+    <th class="center">
+      Mark Done
+    </th>
+    <th class="center">
+      Details
+    </th>
+    <th class="center">
+      Delete
+    </th>
+  </tr>
+  <tbody>
+    <% @all_tasks.each do |task| %>
+      <% if task.completed_date.present? %>
+        <% @task_css = "completed" %>
```

Comment:
Ahh, yeah. I was probably planning on moving that into the controller at some point. Of course now I know that should be in the model anyway.